c2corg_ui/templates/preferences.html

```diff
@@ -37,6 +37,22 @@ <h3 translate class="green-text">Areas</h3>
         </div>
       </div>
 
+      <div class="preferences-langs">
+        <h3 translate class="green-text">langs</h3>
+        <p translate ng-show="prefCtrl.lang_preferences.length == 0">Filter the document languages of your interest by selecting them.</p>
+        <div class="form-group full full-group">
+          <ul class="types">
+            <li ng-repeat="lang in ${default_langs}" ng-click="prefCtrl.updateLanguages(lang)">
+              <div class="checkbox">
+                <input type="checkbox" ng-checked="prefCtrl.lang_preferences.indexOf(lang) > -1">
+                <span>{{lang | translate}} {{lang}}</span>
```

Comment:
Why do you need to display both the translated string and the untranslated one?

Instead of checkboxes, it would make sense to use buttons that would switch colors (grey to orange?) when activated (like for the activities).